src/builtInPDBObjects/headers/Forest.h

```diff
@@ -44,12 +44,22 @@ using std::filesystem::directory_iterator;
 namespace pdb
 {
 
+    typedef struct {
+    
+         int indexID;
+         bool isLeaf;
+         int leftChild;
+         int rightChild;
+         // returnClass will be the vaule to compare while this is not a leaf node
+         float returnClass; 
+    } Node;
+
     class Forest : public Object
     {
     public:
         ENABLE_DEEP_COPY
 
-        pdb::Vector<pdb::Vector<pdb::Handle<pdb::Node>>> forest;
+        Node forest[MAX_NUM_TREES][MAX_NUM_NODES_PER_TREE];
```

Comment:
Are we doing this to make the sizing static, so that we can skip the usage of Handle? Or, is there a different reason?